config/prow/config.yaml

```diff
@@ -777,6 +777,7 @@ tide:
     kubernetes-sigs/go-open-service-broker-client: squash
     kubernetes-sigs/minibroker: squash
     kubernetes-sigs/vsphere-csi-driver: squash
+    kubernetes-sigs/kernel-module-management: squash
```

Comment:
Can we keep an alphabetical order? I would put this line just above `kubernetes-sigs/krew-index`.

Reply:
Sure. Fixing

Reply:
Fixed.